packages/client/test/integration/Resends2.test.ts

```diff
@@ -428,9 +428,10 @@ describe('Resends2', () => {
     })
 
     it('decodes resent messages correctly', async () => {
-        const publishedMessage = Msg({
-            content: fs.readFileSync(path.join(__dirname, '../data/utf8Example.txt'), 'utf8')
-        })
+        const content = isRunningInElectron()
+            ? Buffer.from('Hęłłö Wørłd! 1234 @#$$%^&*() 你好，世界 Привет, мир こんにちは世界', 'utf-8').toString('utf-8')
+            : fs.readFileSync(path.join(__dirname, '../data/utf8Example.txt'), 'utf8')
```

Comment:
We could use the hardcoded data also in NodeJS and remove the `utf8Example.txt`?

Reply:
Actually let's create express server and serve this file up that way in both instances.